scripts/constants.sh

```
@@ -1,11 +1,17 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+
 # Current branch
 current_branch_temp=$(git symbolic-ref -q --short HEAD || git describe --tags --always || echo unknown)
 # replace / with - to be a docker tag compatible
 current_branch=${current_branch_temp////-}
```

Shellcheck (0.10.0)

[warning] current_branch appears unused. Verify use (or export if used externally).

(SC2034)